platforms/documentation/docs/src/docs/userguide/best-practices/best_practices_dependencies.adoc

```diff
@@ -162,6 +162,56 @@ include::sample[dir="snippets/bestPractices/settingsRepositoriesDo/groovy",files
 
 `<<tags_reference.adoc#tag:structuring-builds,#structuring-builds>>`
 
+[[use_content_filtering]]
+== Use Content Filtering when using multiple Repositories
+
+When using multiple repositories, use <<filtering_repository_content.adoc#repository-content-filtering,repository content filtering>> to ensure that dependencies are resolved from an appropriate repository.
+
+=== Explanation
+
+If your build declares more than one repository, you should also declare content filters on these repositories to ensure you search for and obtain dependencies from the correct place.
+The repository that contains the remaining artifacts should come at the end, so we only query it if we don’t find the included modules in the other repositories.
```

Comment:
I think we need to explain here briefly that the repositories are queried in order of set up in the repositories block, and then lead to the suggestion to have no content filter on the "fall back" repository that is at the end of the list.